### PR TITLE
Room leave : Warn the last admin when he try to leave a room

### LINKED ISSUE #262: Room Leave : Show a warning when the last admin of a room try to leave
Show a warning modal when the last administrator of a room try to leave it. 
This is made in order to avoid a room with nobody inside with the invite power.

### PR DESCRIPTION
**Pull request by Yoshiin**
Resolve #262

### CODE REVIEW

**Comment by Yoshiin** [comment]
Do not merge before wording was done.

File: src/Tchap.js [hunk 1]
```diff
@@ -52,6 +52,22 @@ class Tchap {
         const res = await fetch(url, options);
         const json = await res.json();
     }
+
+    static isUserLastAdmin(room) {
+        const userId = MatrixClientPeg.get().getUserId();
+        const members = room.currentState.getMembers();
```
Comment (giomfo) [inline src/Tchap.js]: Please check whether `getMembers()` return only joined members.
I think we should ignore here all other members invited, left, banned...

Comment (Yoshiin) [reply src/Tchap.js]: Yes, I have to change the method used.

File: src/Tchap.js [hunk 2]
```diff
@@ -52,6 +52,22 @@ class Tchap {
...
         const json = await res.json();
     }
+
+    static isUserLastAdmin(room) {
+        const userId = MatrixClientPeg.get().getUserId();
+        const members = room.getJoinedMembers();
+        let adminNumber = 0;
+        let isUserAdmin = false;
+        members.forEach(m => {
+            if (m.powerLevelNorm === 100) {
```
Comment (giomfo) [inline src/Tchap.js]: you should consider level >= 100

**Pull request merged**